### PR TITLE
Add example notebook for graph data input

### PR DESCRIPTION
**Pull request by MisterPNP**
Add example notebook for graph data input

### CODE REVIEW

File: examples/data_profiler_demo.ipynb [hunk 1]
```diff
@@ -743,7 +743,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3.8.13 ('DataProfiler')",
```
Comment (JGSweets) [inline examples/data_profiler_demo.ipynb:746]: let's remove the changes here.

File: examples/graph_data_demo.ipynb [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    "report = profile.report()\n",
+    "\n",
+    "pp = pprint.PrettyPrinter(sort_dicts=False, compact=True)\n",
+    "pp.pprint(report)"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3.8.13 ('DataProfiler')",
```
Comment (JGSweets) [inline examples/graph_data_demo.ipynb:157]: let's manually set this to just `Python 3`

File: examples/graph_data_demo.ipynb [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.13"
+  },
+  "orig_nbformat": 4,
+  "vscode": {
+   "interpreter": {
+    "hash": "a2197c79d714b32f9f9ed41e6865be7abee781a88954d580d283659e068a32d9"
+   }
```
Comment (JGSweets) [inline examples/graph_data_demo.ipynb:177]: remove the vscode section

File: examples/graph_data_demo.ipynb [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.13"
+  },
+  "orig_nbformat": 4,
```
Comment (JGSweets) [inline examples/graph_data_demo.ipynb:173]: this comma likely needs to be removed if vscode removed

File: examples/graph_data_demo.ipynb [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+   "source": [
+    "# GraphData and GraphProfiler pipeline demo"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cab7a569",
+   "metadata": {},
+   "source": [
+    "We have added the ability to load and profile graph datasets to the DataProfiler. Similarly to the rest of DataProfiler profilers, this is split into two components:\n",
```
Comment (JGSweets) [inline examples/graph_data_demo.ipynb:16]: Maybe instead, of the first sentence:
> The Data Profiler can also load and profile graph datasets.

File: examples/graph_data_demo.ipynb [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
```
Comment (JGSweets) [inline examples/graph_data_demo.ipynb:55]: make sure to clear the outputs

**Review by taylorfturner — APPROVED** [review]

**Review by JGSweets — APPROVED** [review]

**Pull request merged**